app/models/spree/shipment.rb

```diff
@@ -159,13 +159,18 @@ def editable_by?(_user)
     end
 
     def manifest
-      inventory_units.joins(:variant).includes(:variant).group_by(&:variant).map do |variant, units|
```

Comment:
did you remove this because the scoping in line 165?

Reply:
This is the merge of decorator with original class from spree. This is the original version that was in spree, but we were already changing it in the decorator.